backend/audit/test_deleted_access.py

```diff
@@ -1,6 +1,28 @@
+from django.contrib.auth.models import User as DjangoUser
 from django.test import TestCase
+
 from model_bakery import baker
-from .models import DeletedAccess
+from .models import (
+    Access,
+    DeletedAccess,
+    SingleAuditChecklist,
+    User,
+    remove_email_from_submission_access,
+)
+
+
+def _make_test_users_by_email(emails: list[str]) -> list[DjangoUser]:
+    return [baker.make(User, email=email) for email in emails]
+
+
+def _make_access(sac: SingleAuditChecklist, role: str, user: DjangoUser) -> Access:
+    return baker.make(Access, user=user, email=user.email, sac=sac, role=role)
+
+
+def _make_accesses(
+    sac: SingleAuditChecklist, role: str, users: list[DjangoUser]
+) -> list[Access]:
+    return [_make_access(sac, role, user) for user in users]
 
 
 class DeletedAccessTests(TestCase):
```

Comment:
nit: should we add a test case that supplies a `removing_user` and/or `removal_event` rather than using the defaults so that we know everything is getting passed along as expected?

Reply:
I added a test for this in https://github.com/GSA-TTS/FAC/pull/2849/commits/8e7c02ace6f214e911bbcdf3da9d5ba478a73187